src/QueryM/JsonWsp.purs

```diff
@@ -76,30 +75,29 @@ type JsonWspResponse (a :: Type) =
   }
 
 -- | A wrapper for tying arguments and response types to request building.
-newtype JsonWspCall :: Type -> Type -> Type
-newtype JsonWspCall (i :: Type) (o :: Type) = JsonWspCall
```

Comment:
I think the task was to remove the `Proxy` parameter from the functions and not the type parameter from `JsonWspCall`

Reply:
That's right, we definitely want to keep the type parameter here

Reply:
sorry, I struggled to understand the idea of `Proxy`. hope it's ok now